channels/candidate-4.3.yaml

```diff
@@ -43,4 +43,6 @@ versions:
 # 4.3.15 Errata: https://access.redhat.com/errata/RHBA-2020:1529
 - 4.3.15
 
+# No 4.3.16 because it was rejected by the release controller
```

Comment:
But mostly because [it had unsigned OVN RPMs][1].

[1]: https://github.com/openshift/cincinnati-graph-data/pull/205#issuecomment-619049150